language/move-prover/bytecode-to-boogie/test_mvir/verify-stdlib/libra_account.mvir

```diff
@@ -697,12 +743,7 @@ module LibraAccount {
         sender_bytes = AddressUtil.address_to_bytes(move(sender));
 
         count_bytes = U64Util.u64_to_bytes(*copy(count));
-        // TODO: the below is supposed to be
-        //   *move(count) = *copy(count) + 1;
-        // However, this creates the obligation to deal with potential even counter overflow in about every
-        // precondition. We need to find some way to turn perhaps overflow checking off if its pragmatically
-        // unnecessary.
-        *move(count) = *copy(count);
+        *move(count) = *copy(count) + 1;
```

Comment:
Is it ok for the counter to overflow?
If so, we could implement "wrap-around" with 
if *copy(count) == U64_MAX-1 then *move(count) = 0 else *move(count) = *copy(count) + 1
[or something like that]

NOTE:  If we find ourselves repeating the same thing in lots of conditions, we should think about notational improvements to allow re-use.

Reply:
notational improvements would be a good idea. I don't know if the counter is allowed to overflow. Right now our Add instructions disallows it.